chapters/infrastructure/linters/reading/README.md

```diff
@@ -1,14 +1,37 @@
-### Linters
+# Linters
 
-Owner: Gabi Mocanu
-Reviewer: Alex Apostolescu
+Linters are tools for checking consistency and quality of code, but they can be used on every type of content.
```

Comment:
I think it is worth mentioning that some linters cover spellchecking too. This ties well with the last part about "every type of content".

Reply:
I added an example section for this.
I will add one more section about rules.